physionet-django/project/forms.py

```diff
@@ -1123,6 +1125,9 @@ def __init__(self, project, requester, template, *args, **kwargs):
         self.project = project
         self.requester = requester
 
+        # Make lay_summary optional in the form
+        self.fields['lay_summary'].required = False
```

Comment:
I think we should make this a required field because it is probably a good thing for everyone.